assets/src/edit-story/masks/output.js

```diff
@@ -81,13 +81,9 @@ export default function WithMask({
       style={{
         ...(fill ? FILL_STYLE : {}),
         ...style,
-        ...(!isBackground
-          ? {
-              clipPath: `url(#${maskId})`,
-              // stylelint-disable-next-line property-no-vendor-prefix
-              WebkitClipPath: `url(#${maskId})`,
-            }
-          : {}),
+        clipPath: `url(#${maskId})`,
+        // stylelint-disable-next-line property-no-vendor-prefix
+        webkitClipPath: `url(#${maskId})`,
```

Comment:
Ok. I'm totally confused. Is it "W" or "w"? Both?

Reply:
It's `WebkitClipPath` but the linter keeps changing it back...

Reply:
Disabled stylelint for that line for now